### PR TITLE
allow to set arbitrary log options in global

### PR DESCRIPTION
**Pull request by hoonetorg**

### CODE REVIEW

File: haproxy/templates/haproxy.jinja [hunk 1]
```diff
@@ -23,8 +23,14 @@
 # Global settings
 #------------------
 global
+{%- if salt['pillar.get']('haproxy:global:log', []) != [] %}
```
Comment (babilen) [inline haproxy/templates/haproxy.jinja]: You could just iterate over `salt['pillar.get']('haproxy:global:log', [])` directly here. If it is empty there is nothing to iterate over.

Comment (hoonetorg) [inline haproxy/templates/haproxy.jinja]: @babilen: good catch if there wasn't the 'else' ;)

**Comment by hoonetorg** [comment]
@babilen:
having such an else could be omitted, if there would be a defaults.yaml/map.jinja as I discussed with @johnkeates here  https://github.com/saltstack-formulas/haproxy-formula/issues/48. 
That would be a larger change.

File: haproxy/templates/haproxy.jinja (hunk 1 above)
Comment (babilen) [inline haproxy/templates/haproxy.jinja]: Right, might make sense to use those entries as default then.

**Comment by kyrias** [comment]
Ping?

**Comment by johnkeates** [comment]
Would be great if me or some other top contributor could actually manage this repo for PR/issues..

**Comment by gtmanfred** [comment]
@johnkeates i have added you to the maintainers list.

Thanks for helping out!
Daniel

**Comment by johnkeates** [comment]
@gtmanfred Thanks, I'll help out where I can! Starting with the formulas I'm familiar with, of course...
 
Quick question: some formulas seem to be 'forked' from their upstreams, while others are copies and have no matched upstream. If there any rule/reason behind this?

**Comment by gtmanfred** [comment]
Usually we just fork it from whoever initially created it.

The ones that aren't forked were created directly in the formulas afaik.

There is no hard and fast rule about it, if people want to transfer the formula over to the repo, i don't see a problem with that.

**Comment by kyrias** [comment]
@johnkeates Ping? :p

**Comment by johnkeates** [comment]
pong!

**Pull request merged**